### PR TITLE
Gym pipeline

### PR DESCRIPTION
**Pull request by kennymckormick**
Add data pipeline for GYM

### CODE REVIEW

File: tools/data/gym/download.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# The code is licensed under the MIT licence.
+import argparse
+import os
+import subprocess
+
+import mmcv
+
+import ssl  # isort:skip
+
+from joblib import Parallel, delayed  # isort:skip
```
Comment (dreamerlin) [inline tools/data/gym/download.py:12]: isort comments seem redundant.

File: docs/changelog.md [hunk 2]
```diff
@@ -6,17 +6,22 @@
...
 - Support PyTorch 1.7 in CI ([#312](https://github.com/open-mmlab/mmaction2/pull/312))
 - Add random seed for building filelists ([#323](https://github.com/open-mmlab/mmaction2/pull/323))
 - Move docs about demo to `demo/README.md` ([#329](https://github.com/open-mmlab/mmaction2/pull/329))
+- Remove redundant code in `tools/test.py` ([#310](https://github.com/open-mmlab/mmaction2/pull/310))
 
 **Bug Fixes**
 - Fix a bug in BaseDataset when `data_prefix` is None ([#314](https://github.com/open-mmlab/mmaction2/pull/314))
 - Fix the bug of HVU object `num_classes` from 1679 to 1678 ([#307](https://github.com/open-mmlab/mmaction2/pull/307))
+- Fix OmniSource training configs ([#321](https://github.com/open-mmlab/mmaction2/pull/321))
+- Fix Issue #306: Bug of SampleAVAFrames ([#317](https://github.com/open-mmlab/mmaction2/pull/317))
```
Comment (dreamerlin) [inline docs/changelog.md:21]: Since u r here, u can mention [this](https://github.com/open-mmlab/mmaction2/pull/307)

Comment (kennymckormick) [reply docs/changelog.md:21]: Seems that this PR is already in changelog.

Comment (dreamerlin) [reply docs/changelog.md:21]: My fault = =

**Review by dreamerlin — APPROVED** [review]

**Comment by kennymckormick** [comment]
@innerlee  PR ready to be merged

**Pull request merged**